App.js

```diff
@@ -0,0 +1,83 @@
+import { StatusBar } from "expo-status-bar";
+import { useState } from "react";
+import { FlatList } from "react-native";
+import {
+  Button,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+  ScrollView,
+} from "react-native";
+import GoalItem from "./components/GoalItem";
+import GoalInput from "./components/GoalInput";
+
+export default function App() {
+  const [modalIsVisible, setModalIsVisible] = useState(false);
```

Comment:
you can rename the state

```suggestion
 const [isModalVisible, setIsModalVisible] = useState(false);
```